draft-ietf-taps-interface.md

```diff
@@ -1337,6 +1337,9 @@ functionality to implement Clone. In that case, entangled Connections are
 multiplexed together, giving them similar treatment not only inside endpoints
 but also across the end-to-end Internet path.
 
+Note that calling Clone() may result in on-the-wire signaling depending on the
+underlying Protocol Stack.
```

Comment:
Maybe we can be even more concrete here, like "... may result in on-the-wire signalling, e.g. to open a new connection,.."?